README.md

```diff
@@ -4,6 +4,35 @@ This repository contains a CMake template for Archicad Add-On Development.
 
 [![Build](https://github.com/GRAPHISOFT/archicad-addon-cmake/actions/workflows/build.yml/badge.svg)](https://github.com/GRAPHISOFT/archicad-addon-cmake/actions/workflows/build.yml)
 
+## Cloning the repository
+
+https://github.com/GRAPHISOFT/archicad-addon-cmake-tools is included as a submodule.
+
+- Clone the repository using
+  ```
+  git clone --recurse-submodules
+  ```
+  **OR**
+  
+  Run this command after the **first time** you clone the repository:
```

Comment:
I prefer keeping this doc as simple as possible, probably it's enough to include the clone and pull commands.

Reply:
Resolved